lib/commanded/event_store/adapter.ex

```diff
@@ -79,7 +79,12 @@ defmodule Commanded.EventStore.Adapter do
 
   @doc """
   Acknowledge receipt and successful processing of the given event received from
-  a subscription to an event stream.
+  a subscription to an event stream. Note that if the event is part of a batch,
+  all events that precede this event in the batch are considered to be acknowledged
+  as well.
+
+  TODO Batching: this holds true for the PostgreSQL implementation, needs to be added to in_memory and
+       verified for EventStoreDB.
```

Comment:
~~Looks like the InMemory implementation is done~~. I think we can go ahead without EventStoreDB as I don't think it's going to see much use going forward.

Reply:
I'd like to see the InMemory adapter implement batching as well before we merge.